assignment-v3-beta/test-case-7.json

```diff
@@ -0,0 +1,2009 @@
+{
+    "experiment": "experiment_with_holdout",
+    "valueType": "string",
+    "subjects": [
+      "6255e1a70560594e38edcb55",
```

Comment:
Personally, I find this list of a 1k subjects to be opaque when it comes to debugging, but these are the same subjects used in `randomization_algo` when the `expectedAssignments` updated to match holdouts

Reply:
Ya this is a big file - can it be accomplished with fewer subjects?